src/lexer.coffee

```diff
@@ -48,6 +48,7 @@ exports.Lexer = class Lexer
     @seenExport = no             # Used to recognize EXPORT FROM? AS? tokens.
     @importSpecifierList = no    # Used to identify when in an IMPORT {...} FROM? ...
     @exportSpecifierList = no    # Used to identify when in an EXPORT {...} FROM? ...
+    @includesCSX = no            # Used to optimize CSX checks.
```

Comment:
Maybe the only thing to clarify, although it should be pretty obvious: This is set for the whole program when we first encounter CSX. It is used so that programs without CSX tags don't get the perf hit from CSX checks.